src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -3808,7 +3813,7 @@ public static boolean isNumericSpace(final CharSequence cs) {
         final int sz = cs.length();
         for (int i = 0; i < sz; i++) {
             final char nowChar = cs.charAt(i);
-            if (nowChar != ' ' && !Character.isDigit(nowChar)) {
+            if (nowChar != PAD && !Character.isDigit(nowChar)) {
```

Comment:
The concept here is not "padding", it's "spacing", so it does not feel appropriate to use the name "PAD".  Note that we have a SPACE constant already but it's a String, not a char, so maybe we need a private space char constant. I'm not sure.

Reply:
HI @garydgregory .You're right., I have the same doubt. For the moment I removed that change